pkg/kn/core/root.go

```diff
@@ -45,7 +45,8 @@ func NewDefaultKnCommand() *cobra.Command {
 	// and will not be parsed
 	pluginsDir, lookupPluginsInPath, err := extractKnPluginFlags(os.Args)
 	if err != nil {
-		panic("Invalid plugin flag value")
+		fmt.Fprintf(os.Stderr, "%s\n", err)
+		os.Exit(1)
```

Comment:
Changed this because there's no reason to show a stack trace to the end user for a bad cmd line arg. Just show the error message.